markhor_flippers/src/markhor_hw_interface_flippers.cpp

```diff
@@ -4,6 +4,31 @@
 
 #include "markhor_hw_interface_flippers.hpp"
 
+static float fr_target = 0;
+static float fl_target = 0;
+static float rr_target = 0;
+static float rl_target = 0;
+
+float MarkhorHWInterfaceFlippers::getFRTarget()
```

Comment:
If you set the variable as public in the class definition, we wouldn't need those getters. That means that the other node would be able to modify them... which is not ideal. What do you think ?

Reply:
If we remove the getters, that remove code, which is ideal.